fluentd/run.sh

```diff
@@ -209,6 +209,10 @@ else
     umount /var/lib/docker/containers/*/shm || :
 fi
 
+if [ "${TRANSFORM_EVENTS:-}" = true ] ; then
+    sed -i 's/\(.*@type viaq_data_model.*\)/\1\n  <event_trans>\n    type eventrouter\n    tag "kubernetes.**"\n  <\/event_trans>/' $CFG_DIR/openshift/filter-viaq-data-model.conf
```

Comment:
nack, according to https://github.com/ViaQ/fluent-plugin-viaq_data_model/pull/9#issuecomment-329044061